service/src/main/java/skills/controller/PublicConfigController.groovy

```diff
@@ -107,6 +111,11 @@ class PublicConfigController {
         if (oAuthProviders) {
             res['oAuthProviders'] = oAuthProviders
         }
+
+        if(authMode.name() == 'SAML2'){
+            String regId = environment.getProperty("spring.security.saml2.registrationId");
```

Comment:
Please use the `@Value` annotation method to inject spring property rather than using `environment.getProperty`

Reply:
Done